_articles/fr/2023-11-07-system-props.md

```diff
@@ -0,0 +1,169 @@
+---
+contentType: article
+lang: fr
+date: 2023-11-07
+slug: system-props
+title: "System props : Flexibilité et Cohérence dans le Design"
+excerpt: Découvrez ce que sont les System Props, quelles sont ses avantages, et comment les intégrer pour améliorer vos composants d'interface.
+categories:
+    - javascript
+keywords:
+    - design system
+    - system props
+    - frontend
+authors:
+    - kdung
+---
+Dans cet article, je vais vous présenter le concept des System Props en précisant ses avantages avec des exemples concrets d'utilisation. Je vais aussi expliquer comment le mettre en place sur vos projets comme nous l'avons fait sur nos projets au sein du [Studio Eleven Labs](https://eleven-labs.com/conception-d-application) à travers l'implémentation d'un [Design System](https://blog.eleven-labs.com/fr/pourquoi-creer-design-system/).
+
```

Comment:
introduire notion de "classe utilitaire" (en précisant qu'on pourrait aussi faire du css in js)